### PR TITLE
Get images from a scanner

### PR DESCRIPTION
**Pull request by beroal**

### CODE REVIEW

File: apparmor.d/profiles-s-z/xsane-gimp [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2024 Roman Beslik <me@beroal.in.ua>
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/3.0>,
+
+include <tunables/global>
+
+@{exec_path} = @{bin}/xsane-gimp
+profile xsane-gimp /{,usr/}{,s}bin/xsane-gimp {
```
Comment (roddhjav) [inline apparmor.d/profiles-s-z/xsane-gimp:10]: Attachment needs to be: `@{exec_path}`

File: apparmor.d/profiles-s-z/xsane-gimp [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  network inet6 dgram,
+  network netlink raw,
+
+  @{exec_path} mr,
+  @{system_share_dirs}/gimp/{,**} r,
+  @{system_share_dirs}/sane/xsane/{,**} r,
+  @{system_share_dirs}/snmp/mibs/{,**} r, # network
+  /etc/sane.d/{,**} r,
+  owner @{HOME}/.sane/{,**} rw,
+  owner @{tmp}/xsane-* rw,
```
[the remark this answers is not in the document]
Comment (beroal) [reply apparmor.d/profiles-s-z/xsane-gimp:27]: Log:

```
apparmor="DENIED" operation="mknod" class="file" profile="xsane-gimp" name="/tmp/xsane-preview-level-0-pixma:04A91761__1118DF.ppm-1001-Vde2Ha"  comm="xsane" requested_mask="c" denied_mask="c" fsuid=1001 ouid=1001 FSUID="user" OUID="user"
apparmor="DENIED" operation="mknod" class="file" profile="xsane-gimp" name="/tmp/xsane-conversion-pixma:04A91761__1118DF.ppm-1001-PIVvsX"  comm="xsane" requested_mask="c" denied_mask="c" fsuid=1001 ouid=1001 FSUID="user" OUID="user"
```

“preview” means a preview scan. “conversion” means a final scan. “1001” is the UID. “pixma” is the printer's name. I have no idea what other fields mean.

[the remark this answers is not in the document]
Comment (roddhjav) [reply apparmor.d/profiles-s-z/xsane-gimp:27]: Then restrict it this way: `owner @{tmp}/xsane-*-@{rand6} rw,`

[the remark this answers is not in the document]
Comment (beroal) [reply apparmor.d/profiles-s-z/xsane-gimp]: Done.

**Comment by roddhjav** [comment]
Thanks, merged.

**Pull request merged**